T/TurboPFor/build_tarballs.jl

```diff
@@ -0,0 +1,51 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "TurboPFor"
+version = v"1.0.0"
```

Comment:
We prefer to use the upstream version, but apparently they don't have any stable release yet, so better to use
```suggestion
version = v"0.0.1"
```
to be conservative